### DISCUSSION TITLE
Status of Flux-schnell LoRA training

### DISCUSSION THREAD

**Discussion by primecai** [bghira/SimpleTuner#1238] [Q&A]
Hi, Just wondering what is the status of FLUX-schnell LoRA training on this repo? I saw some issue mentioning there might be some problems [(here)](https://github.com/bghira/SimpleTuner/pull/803#issue-2472081328) but I am unsure whether it has been updated since.

**Comment by bghira** [comment]
not really a focus here. you can use Shuttle v3 for tuning but it's flaky.

**Comment by bghira** [comment]
fixed schnell training via assistant lora.
